config/schedule.rb

```diff
@@ -1,6 +1,6 @@
-every 5.minutes do
+every 5.minutes, :roles => [:app] => do
```

Comment:
This should read:

```ruby
every 5.minutes, rules: [:app]
```

(using the Ruby 1.9 hash syntax, and removing the extra `=>`)